scripts/ci/update-github

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -ex
+
+TAG_NAME=$(git tag -l --contains HEAD)
+
+rm -rf dcos-nightly
+
+# Setup Git
+git config --global user.email "$GIT_USER@users.noreply.github.com"
+git config --global user.name "MesosphereCI Robot"
+git config credential.helper 'cache --timeout=300'
+git clone "https://$GIT_USER:$GIT_PASSWORD@github.com/mesosphere/dcos" dcos-nightly
+ls -l
+
+cd dcos-nightly
+
+if [[ $TAG_NAME ]]
+then
+  git checkout -- .
+  git remote add upstream "https://$GIT_USER:$GIT_PASSWORD@github.com/dcos/dcos"
+  git checkout master
+  git checkout -b dcos-ui/$TAG_NAME
+
+  cp -fr ../buildinfo.json packages/dcos-ui/buildinfo.json
+
+  git add "packages/dcos-ui/buildinfo.json"
+  git commit -m "chore(dcos-ui): update package"
+  git push --force origin dcos-ui/$TAG_NAME
```

Comment:
this being automated, do we really want to force push here? Shouldn't it fail if there is already is a branch with the same name?

Reply:
Hm, I just think in general it's better to be forceful in this script. Imagine I wanted to update the position of a tag, I'd want this job to do that forcefully, and not fail because it already exists. 

But I don't feel strong about this, and I could imagine arguments for either.